docs/modules/ROOT/pages/documentation/ecosystem/docker-images/micro-image-overview.adoc

```diff
@@ -19,8 +19,6 @@ Keep in mind the following:
 * Accessing the default HTTP endpoint will yield a `404 - Not Found` response.
 * The instance's JVM process runs under the `payara` user.
 
-NOTE: It is always recommended to specify the tag name of the Docker image that you plan to use. Otherwise, you will be running the latest version and might encounter some unwanted effects due to changes between versions.
```

Comment:
This is a best practice and must not be removed!

Reply:
Not really, since:

* Community is not recommended to run in production and we give no guarantees on stability.
* We only give community support on the latest release, so there's no point in telling users to stick with an specific version